sdks/python/apache_beam/runners/dataflow/dataflow_runner.py

```diff
@@ -461,18 +476,21 @@ def _get_side_input_encoding(self, input_encoding):
 
   def _get_encoded_output_coder(self, transform_node, window_value=True):
     """Returns the cloud encoding of the coder for the output of a transform."""
+    output_tag = None if None in transform_node.outputs.keys() else next(
+        iter(transform_node.outputs.keys()))
```

Comment:
I'm only comfortable picking an arbitrary one if there's only one, but that should be doable within the if. 

The other place it's used (for window coder), any is fine, with the comment that all outputs of primitive operations have the same windowing.